src/for-developers/rust-sdk.md

```diff
@@ -0,0 +1,18 @@
+# Rust SDK
+
+Rust SDK for Fuel. It can be used for a variety of things, including but not limited to:
+
+- Compiling, deploying, and testing Sway contracts;
```

Comment:
Compiling is actually not currently supported as an SDK feature (it used to be)!

```suggestion
- Deploying, and testing Sway contracts;
```